graphql_api/mutation/weak_supervisor.py

```diff
@@ -116,29 +140,46 @@ class Arguments:
         project_id = graphene.ID(required=True)
         information_source_id = graphene.ID(required=True)
         labeling_task_id = graphene.ID(required=True)
+        overwrite_default_precision = graphene.Float(required=False)
+        overwrite_weak_supervision = graphene.JSONString(required=False)
 
     ok = graphene.Boolean()
 
     def mutate(
-        self, info, project_id: str, information_source_id: str, labeling_task_id: str
+        self,
+        info,
+        project_id: str,
+        information_source_id: str,
+        labeling_task_id: str,
+        overwrite_default_precision: Optional[float] = None,
+        overwrite_weak_supervision: Optional[Dict[str, float]] = None,
     ):
         auth.check_demo_access(info)
         auth.check_project_access(info, project_id)
         user = auth.get_user_by_info(info)
         pl_manager.create_payload(
             project_id, information_source_id, user.id, asynchronous=False
         )
+        ws_overwrite = None
+        if overwrite_weak_supervision is not None:
+            ws_overwrite = overwrite_weak_supervision
+        else:
+            ws_overwrite = overwrite_default_precision
+
         labeling_task_item = labeling_task.get(project_id, labeling_task_id)
         for information_source in labeling_task_item.information_sources:
```

Comment:
I always thought Run+WS only selects the current function (which was misinformation on my side) but it selects all from the task which doesn't match the description in our docs (which all all)
![image](https://github.com/code-kern-ai/refinery-gateway/assets/91723236/3850613e-e071-44b8-b183-890d8e3cf31d)

Reply:
will inform devrels for documentation update